packages/browser/src/plugins/segmentio/batched-dispatcher.ts

```diff
@@ -91,10 +91,10 @@ export default function batch(
     }, timeout)
   }
 
-  onPageLeave(() => {
-    pageUnloaded = true
+  onPageChange((unloaded: boolean) => {
```

Comment:
Nit: No need to put "boolean" here; that should be part of the fn signature type